README.md

```diff
@@ -627,6 +627,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Requestly](https://requestly.com/) Open-source Chrome Extension to Intercept, Redirect and Mock HTTP Requests. Featuring [Debugger](https://requestly.com/products/web-debugger/), [Mock Server](https://requestly.com/products/mock-server/), [API Client](https://requestly.com/products/api-client/) and [Session Recording](https://requestly.com/products/session-book/).  Redirect URLs, Modify HTTP Headers, Mock APIs, Inject custom JS, Modify GraphQL Requests, Generate Mock API Endpoints, Record session with Network & Console Logs. Create upto 10 rules in Free Tier. Free for open-source.
   * [seotest.me](https://seotest.me/) — Free on-page SEO website tester. 10 free website crawls per day. Useful SEO learning resources and recommendations on how to improve the on-page SEO results for any website regardless of technology.
   * [snippets.uilicious.com](https://snippets.uilicious.com) - It's like CodePen but for cross-browser testing. UI-licious lets you write tests like user stories and offers a free platform - UI-licious Snippets - that allows you to run unlimited tests on Chrome with no sign-up required for up to 3 minutes per test run. Found a bug? You can copy the unique URL to your test to show your devs exactly how to reproduce the bug.
+  * [TestCollab](https://testcollab.com) - A user-friendly test management software, its  [free](https://testcollab.com/pricing) plan includes QA Copilot for AI aided test automation, Jira integration, unlimited projects, and test case import using CSV/XLSx. Signup [here](https://testcollab.io/start_trial).
```

Comment:
QA co pilot is only free till November so we can remove mention of that. Also remove the sign up and pricing link it’s easily found on the site and likely to change with time.

Reply:
Thanks for the feedback @ripienaar, I have made the changes you suggested, please review.